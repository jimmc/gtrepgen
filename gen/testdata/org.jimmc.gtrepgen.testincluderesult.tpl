{{ $result := include "org.jimmc.gtrepgen.includedwithreturn" -}}
Result of include is {{ $result }}.
