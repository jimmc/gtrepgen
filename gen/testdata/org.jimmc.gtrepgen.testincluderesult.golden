This is the included file with a return value.
Result of include is 2468.
